gdpr/models.py

```diff
@@ -99,6 +190,7 @@ class Meta:
         verbose_name = _('legal reason')
         verbose_name_plural = _('legal reasons')
         ordering = ('-created_at',)
+        unique_together = ('purpose_slug', 'source_object_content_type', 'source_object_id')
```

Comment:
Pro tohle musíme vyřešit migrace - jak ji udělat, pokud tam již někdo může mít předchozí automatické (a migrace, na které se zapomělo).

Reply:
Tohle podle mě vyřešíš snadno, necháš vždy jen LegalReson který expiruje/expiroval poslední.

Reply:
To je v pohodě, ale co s těmi automatickými migracemi a jejich návaznostmi?

Reply:
Nic, už jsem si to vyjasnil s Lubošem.